### PR TITLE
14 setup sveltekit env

### LINKED ISSUE #14: Setup Sveltekit env
prettier
pre-commit
eslint

### PR DESCRIPTION
**Pull request by yuval9313**

### CODE REVIEW

File: ui/.gitignore
Comment (DawnOfHell) [inline ui/.gitignore:1]: why diffrent .gitignore for usi? do we want a unified gitingore ot one for each part of the repo? (couppled with Earthfile)

File: ui/README.md
Comment (DawnOfHell) [inline ui/README.md:1]: do we need this?

**Review by DawnOfHell — APPROVED** [review]

File: ui/README.md
Comment (yuval9313) [reply ui/README.md:1]: It's all about good documentation

Comment (yuval9313) [reply ui/README.md:1]: I had planned to remove it but this actually contains concrete info about what you'd do if you will start to work on svelte env so I think a good practice would be to keep it

**Pull request merged**